text/message_verification.go

```diff
@@ -8,7 +8,8 @@ import (
 const (
 	InfoSelfServiceVerification           ID = 1070000 + iota // 1070000
 	InfoSelfServiceVerificationEmailSent                      // 1070001
-	InfoSelfServiceVerificationSuccessful                     // 1060002
+	InfoSelfServiceVerificationSuccessful                     // 1070002
```

Comment:
Since this is in the login UI, please put it in `message_login` and give it a descriptive name